pymbar/mbar_solvers.py

```diff
@@ -497,6 +504,9 @@ def solve_mbar(u_kn_nonzero, N_k_nonzero, f_k_nonzero, solver_protocol=None):
     """
     if solver_protocol is None:
         solver_protocol = DEFAULT_SOLVER_PROTOCOL
+    for protocol in solver_protocol:
+        if protocol['method'] is None:
+            protocol['method'] = DEFAULT_SOLVER_METHOD
```

Comment:
Might this screw things up by placing inconsistent options for the default method?

Reply:
Not sure how, the `solver_protocol` is checked first to see if its `None` and assign the default Protocol, then each `solver_protocol['method]` is checked to see if *that* is `None`, to assign the default Method

Reply:
OK.